### PR TITLE
Dev setup

### PR DESCRIPTION
**Pull request by vujita**
1. Add icons for various supported ides
2. Add file loader to assets to display
3. Use webpack-dev-server with live reloading in dev mode for faster development experience
4. Convert main process to be trans-piled via webpack

### CODE REVIEW

**Comment by vujita** [comment]
> Let's remove all `*-light.png` files.
👍

File: package.json [hunk 1]
```diff
@@ -1,78 +1,83 @@
-{
```
Comment (alanhamlett) [inline package.json]: What's this change, line endings?

Comment (vujita) [reply package.json]: Hmmm, good point, didn't notice that. Let me see if I can clean that up

File: src/containers/index.tsx [hunk 2]
```diff
@@ -1,15 +1,13 @@
 import * as React from "react";
-import ReactDom from "react-dom";
-import { Provider } from "react-redux";
-import { createStore } from "redux";
+import { render } from "react-dom";
```
Comment (alanhamlett) [inline src/containers/index.tsx]: Thanks!

File: src/html/index.html [hunk 3]
```diff
@@ -8,6 +8,5 @@
   </head>
   <body>
     <div id="container"></div>
-    <script src="../dist/main.js"></script>
```
Comment (alanhamlett) [inline src/html/index.html]: This isn't needed anymore?

File: typings/imags.d.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+declare module "*.png";
```
Comment (alanhamlett) [inline typings/imags.d.ts]: Should this be `images`?

File: src/html/index.html (hunk 3 above)
Comment (vujita) [reply src/html/index.html]: This is handled by the webpack html plugin, it will add in the file reference.

File: webpack.renderer.config.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const plugins = [
+  new HtmlWebpackPlugin({
+    template: path.resolve(__dirname, "src", "html", "index.html")
```
Comment (alanhamlett) [inline webpack.renderer.config.js]: So this is a lodash template now that `html-webpack-plugin` adds the webpack build script to?

Comment (vujita) [reply webpack.renderer.config.js]: Not quite, [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin) is a way to make a generic template to add js into. It has a bit of it's own dsl to modify.  It is nice to put it into the output folder too

**Pull request merged**